memory_monitor.py

```
@@ -0,0 +1,157 @@
+                
+                memory_mb = proc.info['memory_info'].rss / 1024 / 1024
+                pytest_processes.append((proc.info['pid'], memory_mb))
+        except (psutil.NoSuchProcess, psutil.AccessDenied):
+            continue
+    
+    if pytest_processes:
+        print(f"\n🧪 実行中のpytestプロセス:")
+        for pid, memory_mb in pytest_processes:
+            print(f"   PID {pid}: {memory_mb:.1f} MB")
+            if memory_mb > 10000:
+                print(f"      🚨 危険: 非常に高いメモリ使用量")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)